spec/models/target_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  latitude   :string
+#  longitude  :string
+#  radius     :float
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  topic_id   :bigint           not null
+#  user_id    :bigint
+#
+# Indexes
+#
+#  index_targets_on_title     (title) UNIQUE
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (topic_id => topics.id)
+#
+require 'rails_helper'
+
+RSpec.describe Target, type: :model do
+  subject { build(:target) }
+
+  describe 'factory' do
+    it { is_expected.to be_valid }
+  end
+
+  describe 'attributes' do
+    it { is_expected.to have_db_column(:title) }
```

Comment:
We can do the same for the rest of the columns
```suggestion
    it { is_expected.to have_db_column(:title).with_options(null: false) }
```